docs/integration_resources/software/nuke.md

```diff
@@ -0,0 +1,29 @@
+# Nuke
+
+## Running Shift in Nuke
+
+Shift provides a `nshift` utility script to properly bind the Shift UI to Nuke. In order to run and open Shift in Nuke, execute the following code block inside the Nuke *Python Script Editor*:
+
+```python
+import sys
+sys.path.append("<path_to_your_shift_folder>")
+sys.path.append("<path_to_your_shift_folder>/thirdparty/python/Lib/site-packages")
+
+from shift.tools import nshift
+nshift.show()
+```
+
+## Python Interpreter Setup
+In Shift it is possible to launch the execution of a workflow via an application's Python interpreter with the [WorkflowProcess](../reference/nodes/workflow/#workflowProcess-node) node. To achieve this for Nuke, set path to its Python interpreter the following environment variables:
+
+`SHIFT_PROCESS_NUKE : path to Nuke’s Python interpreter (python)`

```

Comment:
Maybe we can add an example, like:

If Nuke is installed in the default location, the path should look something like this:
SHIFT_PROCESS_NUKE="C:\Program Files\Nuke15.0v3\python.exe"

Reply:
I See that we are not adding the example path in any other DCCs so ignore this for now.